server/routes/api/index.js

```diff
@@ -0,0 +1,38 @@
+const apiRouter = require('express').Router();
+const { Octokit } = require('@octokit/rest');
+const debug = require('../../debug');
+
+const octokit = Octokit({
+    baseUrl: process.env.GITHUB_API_URL
+});
+
+apiRouter.get('/hell', (req, res) => res.send('123'));
```

Comment:
Looks like we don't need this path, do we?

Reply:
You god damn right.